STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -1015,6 +1023,14 @@ the certificate.  For STAR, the IdO shall use the cancellation interface
 defined in Section 2.3 of {{RFC8739}}. For non-STAR, the certificate revocation
 interface defined in Section 7.6 of {{RFC8555}}) is used.
 
+The ACME account associated with the delegation plays a crucial role in the
+overall security of the presented protocol.  This, in turn, means that in
+delegation scenarios the security requirements and verification associated with
+an ACME account may be more stringent than in traditional ACME, since the
+out-of-band configuration of delegations that an account is authorized to use,
+combined with account authentication, takes the place of the normal ACME
+authorization challenge procedures.
```

Comment:
I'm not sure what "more stringent security requirements" mean here. We're not adding any authentication mechanism to the protocol.

Also, maybe the first sentence of this paragraph should make it clear that we're talking about the account defined on the IdO.

Reply:
Are you suggesting we drop this because it is not actionable?

Reply:
Yes, or else make it more concrete. For example:
> The IdO MUST ensure that each account is associated with the exact policy (`delegation` object) that defines which domain names can be delegated to the account and how. The IdO is expected to use out of band means to pre-register each NDC to the corresponding account.